src/sharding/ShardingManager.js

```diff
@@ -143,10 +143,11 @@ class ShardingManager extends EventEmitter {
   }
 
   /**
-   * Spawns a single shard.
-   * @param {number} [id=this.shards.size] ID of the shard to spawn -
+   * Creates a single shard.
+   * <warn>Using this method is usually not necessary if you use the spawn method.</warn>
+   * @param {number} [id=this.shards.size] ID of the shard to create -
    * **This is usually not necessary to manually specify.**
```

Comment:
```suggestion
   * <info>This is usually not necessary to manually specify.</info>
```
shouldn't this be an info tag?